### PR TITLE
moritz_test_branch

### PR DESCRIPTION
**Pull request by moekappels**
this is a rollback of changes that created a certain file structure for the json object.

### CODE REVIEW

File: src/main/java/de/internetsicherheit/brl/bloxberg/cache/ethereum/ReducedTransObjectArray.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package de.internetsicherheit.brl.bloxberg.cache.ethereum;
+
+import org.web3j.protocol.core.methods.response.Transaction;
+
+import java.util.ArrayList;
+
+public class ReducedTransObjectArray {
+
+    /**
+     * not being used atm. ignore for now
```
Comment (kiview) [inline src/main/java/de/internetsicherheit/brl/bloxberg/cache/ethereum/ReducedTransObjectArray.java:10]: They are used in the method `addTransaction()`.

File: src/main/java/de/internetsicherheit/brl/bloxberg/cache/gui/HistoricDataVisualizer.java [hunk 2]
```diff
@@ -62,8 +62,8 @@ public void start(Stage primaryStage) throws IOException {
         // original: writer = new EthereumWriter(Path.of(OUTPUTDIRECTORY), filename);
         writer = new EthereumWriter(Path.of(OUTPUTDIRECTORY), filename);
         BlockAggregator dbs = initDataBlockSummerizer(filename);
-        TestCenter testCenter = new TestCenter(client, new File(OUTPUTDIRECTORY + "extractedData6.json"));
-        testCenter.generateJsonFile(0, 10000);
+        TestCenter testCenter = new TestCenter(client, new File(OUTPUTDIRECTORY + "transactionsFrom100kBlocks2.json"));
+        testCenter.generateJsonFile(0, 1000);
```
Comment (kiview) [inline src/main/java/de/internetsicherheit/brl/bloxberg/cache/gui/HistoricDataVisualizer.java:66]: As said before, we should refrain from committing changes to this file, if changes are only done for local testing.

File: src/main/java/de/internetsicherheit/brl/bloxberg/cache/gui/TestCenter.java [hunk 3]
```diff
@@ -52,7 +54,7 @@ public void extractJsonObject(int blockNumber, SequenceWriter seqWriter) throws
         ListIterator it = transList.listIterator();
         while (it.hasNext()) {
             seqWriter.write(new ReducedTransObject((Transaction) it.next()));
-
+            //rdoa.addTransaction((Transaction) it.next());
```
Comment (kiview) [inline src/main/java/de/internetsicherheit/brl/bloxberg/cache/gui/TestCenter.java:57]:
```suggestion
```
Don't commit commented-out code.

**Comment by kiview** [comment]
Please add a clearer title and description.
Why do we need to rollback former change, if the PR was never merged?

File: src/main/java/de/internetsicherheit/brl/bloxberg/cache/gui/HistoricDataVisualizer.java (hunk 2 above)
Comment (moekappels) [reply src/main/java/de/internetsicherheit/brl/bloxberg/cache/gui/HistoricDataVisualizer.java:66]: ok then i will change the outputfile back to a standard file everytime before committing.

File: src/main/java/de/internetsicherheit/brl/bloxberg/cache/ethereum/ReducedTransObjectArray.java (hunk 1 above)
Comment (moekappels) [reply src/main/java/de/internetsicherheit/brl/bloxberg/cache/ethereum/ReducedTransObjectArray.java:10]: which is never called.

File: src/main/java/de/internetsicherheit/brl/bloxberg/cache/gui/HistoricDataVisualizer.java (hunk 2 above)
Comment (kiview) [reply src/main/java/de/internetsicherheit/brl/bloxberg/cache/gui/HistoricDataVisualizer.java:66]: You can simply not commit this change. GIT does not require you to commit all changes in your working directory.

File: src/main/java/de/internetsicherheit/brl/bloxberg/cache/ethereum/ReducedTransObjectArray.java (hunk 1 above)
Comment (kiview) [reply src/main/java/de/internetsicherheit/brl/bloxberg/cache/ethereum/ReducedTransObjectArray.java:10]: In this case, don't even add this class to this PR, if it is unused code.

Comment (moekappels) [reply src/main/java/de/internetsicherheit/brl/bloxberg/cache/ethereum/ReducedTransObjectArray.java]: deleted class

File: src/main/java/de/internetsicherheit/brl/bloxberg/cache/gui/HistoricDataVisualizer.java (hunk 2 above)
Comment (kiview) [reply src/main/java/de/internetsicherheit/brl/bloxberg/cache/gui/HistoricDataVisualizer.java:66]: FYI, conversations should be resolved by the reviewer.

**Comment by kiview** [comment]
Should be close this now, or what is up with this?

**Comment by moekappels** [comment]
yes, jsut get rid of it. multiple pull requests just confuse me.

**Comment by kiview** [comment]
So you should have a clear 1:1 mapping between branches and PR, then it works and makes sense.

**Pull request closed without merging**